aiida/engine/processes/calcjobs/manager.py

```diff
@@ -52,41 +56,41 @@ def __init__(self, authinfo, transport_queue, last_updated=None):
         self._loop = transport_queue.loop
         self._logger = logging.getLogger(__name__)
 
-        self._jobs_cache = {}
-        self._job_update_requests = {}  # Mapping: {job_id: Future}
+        self._jobs_cache: Dict[Hashable, 'JobInfo'] = {}
+        self._job_update_requests: Dict[Hashable, asyncio.Future] = {}  # Mapping: {job_id: Future}
         self._last_updated = last_updated
-        self._update_handle = None
+        self._update_handle: Optional[asyncio.TimerHandle] = None
 
     @property
-    def logger(self):
+    def logger(self) -> logging.Logger:
         """Return the logger configured for this instance.
 
         :return: the logger
         """
         return self._logger
 
-    def get_minimum_update_interval(self):
+    def get_minimum_update_interval(self) -> float:
         """Get the minimum interval that should be respected between updates of the list.
 
         :return: the minimum interval
-        :rtype: float
+
```

Comment:
Why keep these lines empty. Could just remove all of these, no? Dead whitespace

Reply:
I just find it more readable to have a blank line before `"""`

Reply:
That's fine, but that is not really what we have been doing so far in the code base. Ideally we have a pre-commit like `pydocstyle` which I have added for some other repo's, but fixing the errors for `aiida-core` has been to much of an undertaking that I haven't done it so far. Would be really good to add it at some point though. Anyway, I won't make you get rid of all those empty lines for this PR I guess.